cmd/ebrelayer/txs/parser.go

```diff
@@ -73,7 +73,7 @@ func EthereumEventToEthBridgeClaim(valAddr sdk.ValAddress, event types.EthereumE
 	witnessClaim.TokenName = event.Name
 	// the nonce from ethereum event is lock burn nonce, not transaction nonce
 	witnessClaim.Denom = ethbridge.GetDenom(networkDescriptor, tokenContractAddress)
```

Comment:
Is there a reason why we want to create a new field CosmosDenom, instead of using Denom field?

Reply:
It is from Brando's comment, a new field will be much clear, Denom for lock and CosmosDemon for burn. 

[Brando753](https://github.com/Brando753) [6 days ago](https://github.com/Sifchain/sifnode/pull/2543#discussion_r829558342)
Should we add a field labeled CosmosDenom in the protofiles rather then mapping the cosmos denom to Symbol?